server/src/rbx_snapshot.rs

```diff
@@ -299,6 +309,17 @@ fn snapshot_imfs_directory<'source>(
     let init_path = directory.path.join(INIT_MODULE_NAME);
     let init_server_path = directory.path.join(INIT_SERVER_NAME);
     let init_client_path = directory.path.join(INIT_CLIENT_NAME);
+    let init_meta_path = directory.path.join(INIT_META_NAME);
+
+    let meta: InitMetaJson = if let Some(ImfsItem::File(file)) = imfs.get(&init_meta_path) {
+        serde_json::from_slice(&file.contents)
+            .map_err(|inner| SnapshotError::InitMetaError {
+                inner,
+                path: file.path.to_path_buf(),
+            })?
+    } else {
+        InitMetaJson::default()
+    };
```

Comment:
I think we can fold this code down even further!

Instead of assigning the struct out here and defaulting to `InitMetaJson`, can we check for the existence of this file later, and then pull properties directly from it onto the snapshot?

Then we can localize the changes to this function to exactly one region, and limit the impact in cases that _don't_ use an `init.meta.json` file.